config/environments/production.rb

```diff
@@ -93,4 +93,10 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  config.logstasher.enabled = true
+  config.logstasher.logger_path = 'log/logstasher.log'
+  config.logstasher.log_level = Logger::INFO
```

Comment:
question: do we want want this level of info in logs for production, and test below? WARN might be all we need for now.....but either way

Reply:
I have inititally set this to info whilst we are early days so that if anything goes wrong we can follow the flow as best we can. As the project progresses im happy to readjust this.